source/projects/create-project-loading-schema-files.txt

```diff
@@ -108,6 +108,11 @@ to generate DDL files from your relational system.
 
       Importing DDL files with Sybase ASE is not supported.
 
+   .. tab:: DB2
+      :tabid: db2-ddl
+
+      Importing DDL files with DB2 is not supported.
```

Comment:
Can you confirm @tomhollander or @alwaysbemark?

Reply:
Confirming, not supported!